core/src/util/builders.cpp

```diff
@@ -117,12 +116,10 @@ void Builders::buildPolygonExtrusion(const Polygon& _polygon, float _minHeight,
         size_t lineSize = line.size();
         sumIndices += lineSize * 6;
         sumVertices += (lineSize - 1) * 4;
-
-        _ctx.numVertices = sumVertices;
     }
```

Comment:
We actually do need `numVertices` in `Builder` context. Reason being, we will have to count the vertices everytime, to make sure the vertex offsets are correct.

Here `numVertices` is not used for `std::vector::reserve` but to determine next vertexOffsets for indices.